core/trino-main/src/test/java/io/trino/sql/planner/TestDynamicFilter.java

```diff
@@ -212,17 +247,22 @@ public void testCrossJoinBetweenDF()
     }
 
     @Test
-    public void testCrossJoinInequalityNoDFWithCast()
```

Comment:
keep the old test too

Reply:
the old test was split into 2 test cases. I can rejoin them into one but there is no point in keeping both.

Reply:
which cases are that?